libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt

```diff
@@ -382,6 +392,8 @@ class MapboxNavigationViewportDataSource(
             cameraForFollowing = followingCameraOptions,
             cameraForOverview = overviewCameraOptions
         )
+
+        Log.d("MapZoomingTest", "MapboxNavigationViewportDataSource.evaluate(). options: $options, resulting viewportData: $viewportData")
```

Comment:
Why do you need to log options? You'll have them in viewportData.

Reply:
That'd be easier to use the same options later when we continue investigation. But agree, that they're not really needed here.